plugins/woocommerce/tests/e2e-pw/tests/merchant/create-post.spec.js

```diff
@@ -0,0 +1,59 @@
+const { test, expect, request } = require( '@playwright/test' );
+const { admin } = require( '../../test-data/data' );
+
+const postTitle = `Post-${ new Date().getTime().toString() }`;
+
+test.describe( 'Can create a new post', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.afterAll( async ( { baseURL } ) => {
+		const base64auth = btoa( `${ admin.username }:${ admin.password }` );
```

Comment:
The `btoa` function is deprecated, as indicated in the description of this function in the source file (buffer.d.ts):
```
        /**
         * Decodes a string into bytes using Latin-1 (ISO-8859), and encodes those bytes
         * into a string using Base64.
         *
         * The `data` may be any JavaScript-value that can be coerced into a string.
         *
         * **This function is only provided for compatibility with legacy web platform APIs**
         * **and should never be used in new code, because they use strings to represent**
         * **binary data and predate the introduction of typed arrays in JavaScript.**
         * **For code running using Node.js APIs, converting between base64-encoded strings**
         * **and binary data should be performed using `Buffer.from(str, 'base64')` and`buf.toString('base64')`.**
         * @since v15.13.0
         * @deprecated Use `buf.toString('base64')` instead.
         * @param data An ASCII (Latin1) string.
         */
        function btoa(data: string): string;
```

As this description states, it would be better to replace it with `buf.toString('base64')` (you can find more info [here](https://nodejs.org/api/buffer.html#buftostringencoding-start-end)).

Reply:
Thank you for highlighting that! Totally missed this one as the comment was hidden. I've pushed a commit to address this.